app/linux/flutter/generated_plugin_registrant.cc

```diff
@@ -0,0 +1,11 @@
+//
```

Comment:
I think you need to sync your branch with the main

Reply:
I've deleted all of these generated files

Reply:
I just did it, it says it's already in sync.